broker/broker-route-manager/pom.xml

```diff
@@ -103,7 +103,7 @@
                                         <limit>
                                             <counter>LINE</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.68</minimum>
+                                            <minimum>65%</minimum>
```

Comment:
❓ What the reason the expected coverage had to be reduced?

Reply:
Oh yeah, I looked into this more.  We're jumping a few versions (patches, then minor) and nothing stood out. The module only has one small class in it. I could to look to add one but most of the lines left uncovered are catches for exceptions. I would have been more concerned if we had to update several files.

Reply:
I was not really asking to add more tests although that is always a good idea 😉 
I was just wondering why you had to reduce the coverage from 68% to 65% just by bumping the plugin version. If it is due to the plugin version then a lot of code will be impacted. Hopefully the new version is much better at checking coverage for the Java 8 streams and lambdas.

Reply:
Yes understood. I was surprised only 2 modules needed to updated. They seem to be really small so not sure if that was the big factor. In both cases the module has a single class in it. They also added Java 9 support in this version, but not sure how it affected streams and lambdas.